### ISSUE TITLE
Chunk debug tool is not working

### ISSUE THREAD

**Issue by Elon61** [gnembon/carpetmod112#161]
I'm probably just being an idiot, but i can't seem to get it to work? i'm pressing start, setting the minimap to follow, but nothing happens (i.e. it stays blank). 
running latest java 8 (build 1.8.0_301-b09), latest carpet (21_07_25), latest carpet client(1.4c), latest liteloader 1.12.2 i grabbed off the jenkins, latest forge (2855), and nothing else. the carpet rule "chunkDebugTool" is enabled and nothing else, totally clean server. bounding boxes / villages also don't seem to work. the other carpet rules appear to work properly. 
Any ideas?

**Comment by skyrising** [comment]
Can you try without Forge? I seem to remember others having issues using the chunk debug tool with that.

**Comment by Elon61** [comment]
Yup, without forge it works. that's something i suppose. any ideas what the issue with forge is? 
Also, do you have documentation for what the colours mean? i recall it being mentioned in passing in some xcom videos i think, but i can't find it, or any other documentation for the matter.
